playgrounds/remix/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "remix-playground",
+  "private": true,
+  "sideEffects": false,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "build": "remix vite:build",
+    "dev": "remix vite:dev",
+    "lint": "",
```

Comment:
If we don't want linting, you can remove this script. Also remove `.eslintrc.cjs` as well as all `eslint` and `@typescript-eslint` related dependencies.

Reply:
Will do.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1887/commits/55c74b1f708bb9d8566d982957f8a55be5973ace)